andela/AndelaEatsUI
# #158378175 Admin view ratings

#### What does this PR do?

- As an admin user, I should be able to view ratings of all meals on a given day

#### Description of Task to be completed?

- implement the UI to enable an admin view all ratings of meals served for the day
- add action creators and thunk
- add a reducer for ratings
- Write tests

#### How should this be manually tested?

- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ft-admin-view-ratings-158378175`
- run these commands `npm run build` and `npm run dev` 
- Log in as an admin, and from the admin dashboard, click on `Ratings` , you should now see the ratings for that day. To view ratings of passed days, click on `Filter` and select a date and click on `Submit`.


#### What are the relevant pivotal tracker stories?
- [#158378175](https://www.pivotaltracker.com/story/show/158378175)

#### Background Context

#### Screenshots (If Applicable)

<img width="1656" alt="screen shot 2019-01-28 at 10 00 42" src="https://user-images.githubusercontent.com/35847046/51819569-f7185b80-22e3-11e9-84ce-0b3fa289c599.png">
<img width="1676" alt="screen shot 2019-01-28 at 10 02 58" src="https://user-images.githubusercontent.com/35847046/51819570-f7b0f200-22e3-11e9-8797-ce91e032eeb0.png">
